weblate/auth/permissions.py

```diff
@@ -281,13 +281,13 @@ def check_edit_approved(user: User, permission: str, obj: Model):
         ):
             return Denied(
                 gettext(
-                    "Only reviewers can change approved strings, please add a suggestion if you think the string should be changed."
+                    "Only reviewers can change approved strings. Please add a suggestion if you think the string should be changed."
                 )
             )
     if isinstance(obj, Translation):
         component = obj.component
         if obj.is_readonly:
-            return Denied(gettext("The translation is read only."))
+            return Denied(gettext("The translation is read-only."))
```

Comment:
```suggestion
            return Denied(gettext("This is a read-only translation."))
```
Better language, but not consistent form.